crates/sui-core/src/consensus_validator.rs

```diff
@@ -89,11 +92,12 @@ impl TransactionValidator for SuiTxValidator {
         let cert_count = cert_batch.len();
         let ckpt_count = ckpt_batch.len();
         let epoch_store = self.epoch_store.clone();
+        let jwk = self.google_jwk_as_bytes.clone();
         Handle::current()
             .spawn_blocking(move || {
                 epoch_store
                     .signature_verifier
-                    .verify_certs_and_checkpoints(cert_batch, ckpt_batch)
+                    .verify_certs_and_checkpoints(cert_batch, ckpt_batch, jwk)
```

Comment:
I think we could live with non-deterministic signing due to each validator having a different view of the JWK, at least while we are prototyping this.

However, I don't think we can ship non-determinism in the consensus validator, as it may cause a consensus fork. There may be no way around implementing the JWK voting for now. However, doing so should not be terribly difficult either.

Reply:
Actually this may be even more tricky - the JWK votes can only be counted in consensus handler, but validator runs before the commit, and hence I don't think can depend on state computed from consensus.

@andll any thoughts?

Reply:
Ok, I actually found thread about it: 
https://mysten-labs.slack.com/archives/C04FG4Q7YJ3/p1679942005356859

TLDR - the only safe way I can think of if we commit to the signature in the certificate, and therefore do not verify it in the consensus anymore. Now that we are in mainnet this will require some amount of work(thought not impossible)...